lib/wallet/integration/src/Test/Integration/Scenario/API/Shared/Transactions.hs

```diff
@@ -2295,6 +2297,298 @@ spec = describe "SHARED_TRANSACTIONS" $ do
                 , expectField #certificates
                      (`shouldBe` [ delegatingCert3 stakeKeyDerPathParty2])
                 ]
+
+    it "SHARED_TRANSACTIONS_DELEGATION_02 - \
+       \Emulating multi-delegation using shared wallets" $ \ctx -> runResourceT $ do
+
+        -- creating empty parent Shelley wallet
```

Comment:
I suppose we don't really need parent Shelley wallet as Shared wallets could be financed directly from `fixtureWallet`.

Reply:
of course, shared wallet can be parent wallet also. Just wanted to demonstrate that using the same mnemonic but different account indices shelley wallet can multi-delegate. If anyone wants to provide the api in shelley/shared the logic is here in this test